.eslintrc.js

```diff
@@ -178,6 +178,7 @@ module.exports = {
         'aws-toolkits/no-console-log': 'error',
         'aws-toolkits/no-json-stringify-in-log': 'error',
         'aws-toolkits/no-printf-mismatch': 'error',
+        'aws-toolkits/no-async-in-foreach': 'error',
```

Comment:
should we just ban `forEach`? it is kind of pointless.

Reply:
Yea, I think banning it out right has a few advantages. Additionally, determining if a function is async is definitely non-trivial. This work is moved to https://github.com/aws/aws-toolkit-vscode/pull/6281